eng/ci/templates/jobs/build-artifacts.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+  - name: SkipBuildArtifacts
```

Comment:
Minor nit: I prefer booleans in the positive form, not negative. IE: `IncludeArtifacts true/false`, with a default of `true`.

Reply:
I will update in a followup PR. This PR was mainly to get the code-mirror in but I also kept other pipelines to speed up the work.